application/testing/CMakeLists.txt

```
@@ -1001,6 +1001,7 @@ f3d_test(NAME TestInteractionZoomToggleOrthographicProjection DATA cow.vtp INTER
 f3d_test(NAME TestInteractionRotateCameraMinus90 DATA f3d.glb INTERACTION)
 f3d_test(NAME TestInteractionRotateCamera90 DATA f3d.glb INTERACTION)
 f3d_test(NAME TestInteractionRollCameraRotation DATA f3d.glb ARGS -g INTERACTION) #1;4;LeftMouse;MouseMovements
+f3d_test(NAME TestInteractionElevationCameraRotation DATA f3d.glb ARGS -g INTERACTION) #1;4;LeftMouse;MouseMovements
```

```suggestion
f3d_test(NAME TestInteractionElevationCameraRotation DATA f3d.glb ARGS -g INTERACTION) #2;8;LeftMouse;MouseMovements
```

please update the comment to reflect the content of the interaction script

---

are these the digits from the log? it should be 1;8?

KeyPressEvent 2113 316 0 56 1 8 0
CharEvent 2113 316 0 56 1 8 0
KeyReleaseEvent 2113 316 0 56 1 8 0

---

Yes indeed! just:

`#8;8;2`